CHANGELOG-4.0.md

```diff
@@ -1,5 +1,8 @@
 # [4.0.0-beta.2](https://github.com/phalcon/cphalcon/releases/tag/v4.0.0-beta.2) (2019-xx-xx)
 
+# Added
+- Added `Phalcon\Session\Adapter\Stream::getPath()` method that returns current save path.
```

Comment:
The PR comment says that proposed changes fixes the bug described in #14265. Should we mention this as well?

Reply:
Absolutely. Will add this